### DISCUSSION TITLE
code-review-test/

### DISCUSSION THREAD

**Discussion by giscus** [Hyeon9mak/Hyeon9mak.github.io#175] [Blog comments]
# code-review-test/

🖋️ 변별력을 200% 더 높인 개발자 채용 방법: 코드 리뷰 테스트 - 전종욱님  

https://hyeon9mak.github.io/code-review-test/

**Comment by Gomding** [comment]
크.. 코드리뷰 테스트 흥미롭네요 👍
깔끔한 정리 감사합니다~
